reflex/constants.py

```diff
@@ -8,12 +8,16 @@
 from types import SimpleNamespace
 from typing import Any, Type
 
+from platformdirs import PlatformDirs
```

Comment:
When i run this on windows:

```console
ModuleNotFoundError: No module named 'platformdirs'
```

Need to add `platformdirs` to `pyproject.toml` and re-lock deps.